src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const malePerson = people.filter(person => {
+    return century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm';
```

Comment:
```suggestion
    return person.sex === 'm' && (
      !century
      || Math.ceil(person.died / 100) === century
    );
```
imho it's better. change also in other functions

Reply:
+

Reply:
got it, thanks!